db/migrations/0011.add_bluejeans_table.py

```diff
@@ -0,0 +1,42 @@
+#
+# file: migrations/0011.add_bluejeans_table.py
+#
+from yoyo import step
+
+__depends__ = {'0010.add_fkc_course_usage_table'}
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS bluejeans_meetings
```

Comment:
We've been using singular table names, so could we change this to `bluejeans_meeting`?